obs-server.sls

```diff
@@ -1,4 +1,21 @@
-install obs server packages:
+/etc/apt/sources.list:
```

Comment:
I don't think we want to do that. We should instead add the repo into 
/etc/apt/sources.list.d/ instead

Reply:
and you should use https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkgrepo.html

Reply:
OK. I am looking into it

Reply:
ping?